airflow/providers/yandex/hooks/yandex.py

```diff
@@ -122,7 +127,8 @@ def __init__(
         self.connection = self.get_connection(self.connection_id)
         self.extras = self.connection.extra_dejson
         credentials = self._get_credentials()
-        self.sdk = yandexcloud.SDK(user_agent=self.provider_user_agent(), **credentials)
+        endpoint = self._get_field("endpoint", False)
+        self.sdk = yandexcloud.SDK(user_agent=self.provider_user_agent(), endpoint=endpoint, **credentials)
```

Comment:
We need to tests this.
I guess in case of missing endpoint `self._get_field("endpoint", False)` will return `False`

Reply:
thanks for the comment! 
False is OK because here we initialize yandexcloud.SDK that sets default API endpoint in case it's missing:
https://github.com/yandex-cloud/python-sdk/blob/master/yandexcloud/_channels.py#L24

we didn't set endpoint until now, so in these tests no endpoint is passed into hooks  https://github.com/apache/airflow/blob/main/tests/system/providers/yandex/example_yandexcloud.py

Reply:
But it not missing in our case, we would provide `False` in this case and it would evaluate to this wrong value

```python
kwargs_no_endpoint = {"foo": "bar"}
print(kwargs_no_endpoint.get("endpoint", "api.cloud.yandex.net"))

kwargs_none_endpoint = {"foo": "bar", "endpoint": None}
print(kwargs_none_endpoint.get("endpoint", "api.cloud.yandex.net"))

kwargs_false_endpoint = {"foo": "bar", "endpoint": False}
print(kwargs_false_endpoint.get("endpoint", "api.cloud.yandex.net"))
```

Reply:
I guess potentially in other places `self._get_field("foo-bar", False)`produced incorrect behaviour of Yandex Cloud SDK

Reply:
> But it not missing in our case, we would provide `False` in this case and it would evaluate to this wrong value
> 
> ```python
> kwargs_no_endpoint = {"foo": "bar"}
> print(kwargs_no_endpoint.get("endpoint", "api.cloud.yandex.net"))
> 
> kwargs_none_endpoint = {"foo": "bar", "endpoint": None}
> print(kwargs_none_endpoint.get("endpoint", "api.cloud.yandex.net"))
> 
> kwargs_false_endpoint = {"foo": "bar", "endpoint": False}
> print(kwargs_false_endpoint.get("endpoint", "api.cloud.yandex.net"))
> ```

very tru, thanks, force pushed the change, now default endpoint is set in provider